### PR TITLE
Updating env

### PR DESCRIPTION
**Pull request by BearTS**

### CODE REVIEW

**Comment by maisans-maid** [comment]
Good

**Comment by maisans-maid** [comment]
`options.connector` is no longer needed and is just a redundant piece of code now, so might as well remove that from the logic

`options.connector` was supposed to be the mongo uri, that if you provided it, it will not check the password anymore and will just connect to the database.

```js
//checks if options.connector property is supplied 
 if (typeof options.connector === 'string'){ 
      this.connector = options.connector; // use this as a uri if it exist
    } else { // use the password instead if it doesn't
      this.connector = `mongodb://botAdmin:${options.password}@botdev-....`; // more lines here
    };

```

** **

From: [Your comment](https://github.com/maisans-maid/Mai/pull/76#issuecomment-757521133) on (#76)
> Since everyone has different database Connection URI, its better to specify the uri in the environment variable instead of password

options.connector resolves that issue, but then this approach might be better. I will not merge this PR yet as we have to remove  options.connector from the entire logic.

File: config.js [hunk 1]
```diff
@@ -74,7 +74,7 @@ const settings = {
   database: {
     enable: true,
     connector: null,
```
Comment (maisans-maid) [inline config.js]: Remove connector from this logic. It function just the same as the uri

**Comment by BearTS** [comment]
yes indeed, i didnt saw it earlier, i just did this pr real quick because someone who was self hosting wasn't able to figure out what had to be done. I will Surely remove it.

File: struct/Mongoose.js [hunk 2]
```diff
@@ -19,7 +19,7 @@ module.exports = class Mongoose{
     if (typeof options.connector === 'string'){
```
Comment (maisans-maid) [inline struct/Mongoose.js]: remove the entire logic here and just assign `this.connector` to `options.uri`

**Review by maisans-maid — CHANGES_REQUESTED** [review]

**Review by BearTS — COMMENTED** [review]
👍

**Review by maisans-maid — APPROVED** [review]

**Pull request merged**